lib.js

```diff
@@ -30,9 +30,17 @@ export function parseFilename (filename) {
 }
 
 function excerpt (content) {
-  let excerpt_ = content.slice(0, 100)
+  let start = 0
+  if (content.slice(0, 3) === '---') {
+    const secondMark = content.indexOf('---', 3)
+    start =
+    secondMark > 0
+      ? secondMark
+      : 0
+  }
+  let excerpt_ = content.slice(start, 100)
   if (excerpt_.indexOf('\n') > 0) {
```

Comment:
Does this account for cases where there's an empty newline after the second `---`?

e.g.

```
---
hello: world
---

# My content starts here
```